### DISCUSSION TITLE
Pull from Data Sources automatically - possible?

### DISCUSSION THREAD

**Discussion by dcfnet** [code4recovery/12-step-meeting-list#645] [Q&A] [answered]
Hello Ninjas!

We have configured a Data Source. We have successfully pulled in the feed. Everything works as expected. We can refresh the feed at intervals to download changes by manually initiating them. Again, all of that works perfectly! Excellent.

The only question is: Can this be set to refresh automatically? For instance 1X or 2X per day, or even 1X/Week without manual intervention.

Thank you for your attention and cheerful service!
-
Your Drunk District-14 DCM
David
(757) 652-1711

**Comment by brianw-area24** [comment]
Unfortunately, this is not possible at this time. This is a feature that 2 or 3 of us have tried at different times, and each time, we run into major roadblocks that prevent us from doing this. When I first started working on the TSML plugin, having the feeds refresh automatically was at the top of my wish list, but it turned into a much bigger change then I expected, but it was a good way for me to learn more about how WordPress and the TSML plugin work.

One of our members is working on something we hope to integrate into the plugin that would notify the site admin when a feed changes, so they'd know when to go in an do a manual refresh of the feed. At this point I'm not sure when this will be released, but we're really excited about this as a step forward in handling feeds.

Brian W.

**Comment by dcfnet** [comment, marked as answer]
Thank you so much for the reply! The plugin keeps improving and moving forward. I think the work you doing is excellent!

We will refresh the data feed manually at regular intervals. No worries. Thanks again!
-
Your Drunk District-14 DCM
David
(757) 652-1711
